config/custom-environment-variables.json

```diff
@@ -40,6 +40,7 @@
     }
   },
   "application": {
+    "chunkSize": "CHUNK_SIZE",
```

Comment:
Specify "__format" for this end as it number - it may cause problem later when reading it as string by default, example:
![image](https://github.com/user-attachments/assets/5d751fc9-3ff8-476c-93dc-27e6f64216f6)

Reply:
btw, same with "AGGREGATION_MAX_DECIMAL_DIGITS"